lib/src/components/NxTabPanel/NxTabPanel.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useContext} from 'react';
+import classnames from 'classnames';
+
+import { ActiveTabContext } from '../NxTabs/NxTabs';
+
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+const NxTabPanel = function NxTabPanelElement(props: Props) {
+  const {activeTab} = useContext(ActiveTabContext);
+  const {index, className, ...attrs} = props;
+
+  if (activeTab !== index) {
+    return null;
+  }
+
+  return (
+    <div role="tabpanel"
+         className={classnames('nx-tab-panel', className)}
```

Comment:
`.nx-tab-panel` doesn't currently do any styling.  I think we avoid making class names that don't actually do anything.  That said, I think it would be a good idea to at least give it some padding.